extensions/amp-subscriptions/0.1/entitlement.js

```diff
@@ -119,7 +128,18 @@ export class Entitlement {
     const products = json['products'] || [];
     const subscriptionToken = json['subscriptionToken'];
     const loggedIn = json['loggedIn'];
-    return new Entitlement(source, raw, /* service */ '',
-        products, subscriptionToken, loggedIn);
+    const meteringData = json['metering'];
+    let metering = null;
+    if (meteringData) {
+      metering = {
+        quotaLeft: meteringData['quotaLeft'],
```

Comment:
What do you think about making them shorter? `left`, `total` still seem very readable to me.

Reply:
Done